rosetta-lang/model/RosettaExpression.xcore

```diff
@@ -406,3 +407,16 @@ class MinOperation extends ComparingFunctionalOperation, ListOperation {
 class MaxOperation extends ComparingFunctionalOperation, ListOperation {
 }
 
+
+class SwitchOperation extends RosettaUnaryOperation {
+	contains CaseStatement[] values opposite switchOperation
+	contains RosettaExpression ^default
+}
+
+class CaseStatement {
```

Comment:
Thinking about the names a bit - what do you think about `SwitchCase` instead of `CaseStatement`? -- considering we don't have anything else called "statement" in Rune.